### PR TITLE
Implements functionality to unset multiple commands in one call

### PR DESCRIPTION
**Pull request by boonwj**
## Description

This PR implements the functionality to unset multiple commands in a single call.

Any failures encountered during the process will be printed but does not stop the operation.

## Fixes/Implements: # (issue)
#141 

## New dependencies introduced?
N.A.

## Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the [changelog draft](https://github.com/gopinath-langote/1build/blob/master/docs/CHANGELOG.md) for corresponding release
- [x] I have update README (If needed)

### CODE REVIEW

File: cmd/unset.go [hunk 1]
```diff
@@ -38,21 +38,21 @@ This will update the current project configuration file.`,
...
 		if err != nil {
 			utils.PrintErr(err)
 			return
 		}
 
-		index := indexOfCommandIfPresent(configuration, commandName)
-		if index == -1 {
-			utils.Println("Command '" + commandName + "' not found")
-			return
+		for _, commandName := range args {
```
Comment (gopinath-langote) [inline cmd/unset.go]: Current Implementation will print `Not found` message for all missing commands.

how about making a single message with list of commands.
ex. 
```markdown
Following command(s) not found: cmd1, cmd2
```

OR
```markdown
Following command(s) not found:
  - cmd1
  - cmd2
```

Preferably with red-colored output.

WDYT about above options?

File: testing/fixtures/command_unset_fixtures.go [hunk 2]
```diff
@@ -92,3 +94,65 @@ func unsetShouldFailWhenConfigurationFileIsInInvalidFormat(feature string) Test
 		},
 	}
 }
+
+func shouldUnsetMultipleCommands(feature string) Test {
```
Comment (gopinath-langote) [inline testing/fixtures/command_unset_fixtures.go]: Let's also have a test for comments not being empty after removing one of the commands.

ex.
```yaml
project: Sample Project
commands:
  - build: go build
  - test: go test
  - lint: go lint
```

```bash
1build unset test missing1 build missing2 
```

After command:
```yaml
project: Sample Project
commands:
  - test: go test
```

AFAIK, we are not testing this flow.

Comment (boonwj) [reply testing/fixtures/command_unset_fixtures.go]: Yep that sounds good, I will add this test case in.

But to clarify, in the example the final output should be the following right? Since build and test is unset.

```
project: Sample Project
commands:
  - test: go test
```

File: cmd/unset.go (hunk 1 above)
Comment (boonwj) [reply cmd/unset.go]: That makes sense. Both versions are okay, though I am leaning slightly towards the one line version. Do you have a preference for either?

Comment (gopinath-langote) [reply cmd/unset.go]: One liner LGTM.

File: testing/fixtures/command_unset_fixtures.go (hunk 2 above)
Comment (gopinath-langote) [reply testing/fixtures/command_unset_fixtures.go]: Yes. you are right. Updated comment.

**Comment by boonwj** [comment]
Made the following changes based on feedback

- Add additional test to cover identified edge case
- Change error message format 
- Update previous test case based on new error message format
- Add check to ensure that config is only rewritten when there are changes after the unset operation

**Comment by gopinath-langote** [comment]
LGTM so far.

Could you please also - Change readme in `unset` section to address one/more commands to unset and it's behavior.

**Comment by gopinath-langote** [comment]
Also,
Change the command short/long description.
```Frictionless way of managing project-specific commands

Usage:
  1build [flags]
  1build [command]

Available Commands:
  init        Create default project configuration
  list        Show all available commands from the current project configuration
  set         Set new or update the existing command in the current project configuration
  unset       Remove the existing command in the current project configuration

Flags:
  -h, --help      help for 1build
      --version   version for 1build

Use "1build [command] --help" for more information about a command.
```

From
```  unset       Remove the existing command in the current project configuration```

to 
```unset          Remove one or more existing command(s) in the current project configuration```
